.github/workflows/action-test.yml

```diff
@@ -5,6 +5,7 @@ on:
   push:
     branches:
     - master
+    - reinstate_GAtest_OSX
```

Comment:
This line should be removed before merging, correct?

Reply:
Yes, good catch!

Reply:
Instead of adding branches to the workflow, I propose that we use a PR on master as a trigger to run tests.

Reply:
Yep, good point!  👍